src/python/grpcio_tests/tests/interop/client.py

```diff
@@ -89,8 +101,27 @@ def _create_call_credentials(args):
 def get_secure_channel_parameters(args):
     call_credentials = _create_call_credentials(args)
 
-    channel_opts = None
-    if args.use_tls:
+    channel_opts = ()
+    if args.grpc_test_use_grpclb_with_child_policy:
+        channel_opts += ((
+            "grpc.service_config",
+            '{"loadBalancingConfig": [{"grpclb": {"childPolicy": [{"%s": {}}]}}]}'
+            % args.grpc_test_use_grpclb_with_child_policy),)
+    if args.custom_credentials_type is not None:
+        if args.custom_credentials_type == "compute_engine_channel_creds":
+            assert call_credentials is None
+            google_credentials, unused_project_id = google_auth.default(
+                scopes=[args.oauth_scope])
+            call_creds = grpc.metadata_call_credentials(
+                google_auth.transport.grpc.AuthMetadataPlugin(
+                    credentials=google_credentials,
+                    request=google_auth.transport.requests.Request()))
+            channel_credentials = grpc.compute_engine_channel_credentials(
+                call_creds)
```

Comment:
optional: This combination of API worths to be an example. I could spend hours to find the right way to create the new credentials.

Reply:
The only intended users of this are going to be the cloud client library authors, who we'll be working with directly. I don't think it's desirable to advertise this in general, since it's only meant to be consumed by higher-level auth libraries. If we add an example, it should show how to enable directpath using the google auth library, but that won't be possible until after they've implemented their part.

Reply:
You are right. I missed the point that this is for directpath.